routes/api/users.js

```diff
@@ -2,8 +2,12 @@ import express from 'express';
 import UserValidation from '../../server/middlewares/UserValidation';
 import UserController from '../../server/controllers/UsersController';
 
-const { validateUserSignUp, checkExistingEmail } = UserValidation;
-
+const {
+  validateUserSignUp,
+  checkExistingEmail,
+  validateUserLogin
+} = UserValidation;
+const { userLogin } = UserController;
```

Comment:
For consistency, if you are destructuring this then do the same for signup on line 22.

Reply:
Should I make this change on this pull request? it was introduced during the signup feature implementation.

Reply:
Please, do that here.